pkg/service/service.go

```diff
@@ -63,6 +81,15 @@ func (b *Service) Configure(opts ...options.Option) error {
 	}
 	b.Identity = base58.Encode(pubkey)
 	log.Println("Identity: ", b.Identity)
+
+	if b.SubNats == nil {
+		return nil
+	}
+
+	if js, ok := b.SubNats.(JetStreamer); ok {
+		b.js, err = js.JetStream(nats.Context(b.Context))
```

Comment:
`err` seems to be unhandled, is it fine?

Reply:
Good catch!